shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -238,6 +253,27 @@ - (instancetype)initWithProject:(nullable FlutterDartProject*)project {
   return self;
 }
 
+- (instancetype)initWithEngine:(nonnull FlutterEngine*)engine
+                       nibName:(nullable NSString*)nibName
+                        bundle:(nullable NSBundle*)nibBundle {
+  NSAssert(engine != nil, @"Engine is required");
+  self = [super initWithNibName:nibName bundle:nibBundle];
+  if (self) {
+    if (engine.viewController) {
+      NSLog(@"The supplied FlutterEngine %s is already used with FlutterViewController "
```

Comment:
`%@` will avoid the need for `UTF8String`

Reply:
Oh, neat, thanks.  Fixed.